pkg/cmd/server/api/v1/types.go

```diff
@@ -932,3 +935,19 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string `json:"pluginOrderOverride,omitempty"`
 }
+
+// PodLimitRequestConfig is the configuration for the PodLimitRequest
+// admission controller which overrides user-provided container request/limit values.
+type PodLimitRequestConfig struct {
```

Comment:
Since this only enforces ratios, once that day comes could we just modify this controller to give pod req/limit the same treatment? Then the name would remain accurate and there would be no config changes. For density purposes we really only get to look at scheduling pods...